formats/placeholder/frontend/schema.json

```diff
@@ -0,0 +1,146 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "base_path",
+    "format",
+    "locale"
+  ],
+  "properties": {
+    "base_path": {
+      "type": "string"
+    },
+    "title": {
+      "type": "string"
+    },
+    "description": {
+      "type": "string"
+    },
+    "format": {
+      "type": "string",
+      "pattern": "^(placeholder|placeholder_.+)$",
+      "description": "Should be of the form 'placeholder_my_format_name'. 'placeholder' is allowed for backwards compatibility."
```

Comment:
I may be wrong, but 'placeholder' isn't just there for backwards compatibility AFAIK. It's just as valid as 'placeholder_FORMAT'. The main difference is that the latter allows the content-register to identify the content and add it to the register. I can see situations where we may want to push placeholder content out without this happening. @alext?